gef-ui/server/routes/index.js

```diff
@@ -30,6 +30,7 @@ const confirmCoverStartDate = require('./confirm-cover-start-date');
 const portalActivities = require('./application-activities');
 const cloneGefDeal = require('./clone-gef-deal');
 const unissuedFacilities = require('./unissued-facilities');
+const { default: bankReviewDate } = require('./bank-review-date');
```

Comment:
can you not do `const bankReviewDate = require('./bank-review-date');` as it's a default export?

Reply:
nah default exports do weird stuff

Reply:
Well they export as 
```
{
  default: bankReviewDateThing
}
```

but importing does weird stuff and unwraps that
```
import bankReviewDate from './bank-review-date'
```
whereas commonjs requires do the equivalent of
```
import * as bankReviewDate from './bank-review-date'
```
and dont do any unwrapping shenangigans to hide the default bit.